mods/ra/maps/allies-08a/allies08a.lua

```diff
@@ -0,0 +1,192 @@
+--[[
+   Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+AlliedBoatReinforcements = { "dd", "dd" }
+TimerTicks = DateTime.Minutes(21)
+ObjectiveBuildings = { Chronosphere, AlliedTechCenter }
+ScientistType = { "chan", "chan", "chan", "chan" }
```

Comment:
Something must have gone wrong here while squashing/rebasing. The change here https://github.com/Smittytron/OpenRA/pull/1/files#diff-fedb75b398831c487136300f93364597R12 was not carried over. This needs to be `ScientistTypes`.